api/konnect/v1alpha1/konnect_cloudgateway_dataplane_configuration_types.go

```diff
@@ -196,7 +197,7 @@ type ConfigurationDataPlaneGroupAutoscaleStatic struct {
 // KonnectCloudGatewayDataPlaneGroupConfigurationStatus defines the observed state of KonnectCloudGatewayDataPlaneGroupConfiguration.
 // +apireference:kgo:include
 type KonnectCloudGatewayDataPlaneGroupConfigurationStatus struct {
-	KonnectEntityStatusWithControlPlaneRef `json:",inline"`
+	konnectv1alpha2.KonnectEntityStatusWithControlPlaneRef `json:",inline"`
```

Comment:
Couldn't we just leave the `KonnectEntityStatusWithControlPlaneRef` defined in the `v1alpha1` package here and use it? 🤔 

This way we wouldn't have to import `github.com/kong/kubernetes-configuration/v2/api/konnect/v1alpha2`.

Same applies elsewhere.

Reply:
Ok, I see.

Even though I do believe that "a little copying is better than little dependency" [link](https://go-proverbs.github.io/) [link_yt](https://www.youtube.com/watch?v=PAAkCSZUG1c&t=568s) I do see that this would require more work now for little benefit (given that `v1alpha1` is going away and that Go is not our primary interface for end users).

I'm OK leaving this as is then. 👍